Readme.md

```diff
@@ -1,5 +1,7 @@
 # Chain Core Developer Edition
 
+Chain Core is software designed to operate and participate in permissioned blockchain networks. Each network maintains a cryptographically-secured multi-asset shared ledger. Using this ledger, participants can issue digital assets directly to custodians, who can then transfer them to each other in real time with no transactional intermediary. Each Chain Core holds a copy of the ledger and independently validates each update. A federation of block signers ensures global consistency of the ledger. Digital assets share a common, interoperable format and can represent any units of value that are guaranteed by a trusted issuer — such as currencies, bonds, securities, IOUs, or loyalty points.
```

Comment:
maybe:

Once issued, assets are held in accounts and may be transferred among them in real time with no transactional intermediary. Each issuance or transfer is recorded in the ledger with a cryptographically signed transaction...